packages/otelbin-validation/src/assets/supported-distributions.json

```diff
@@ -261,5 +261,36 @@
       }
     ],
     "ignoredReleases": "<v0.36.0"
+  },
+  "splunk": {
```

Comment:
Please note that this is an auto-generated file. Upon the next job run, these changes will be overwritten.

You have to extend the CI job definition for automatic distribution version discovery to get…

 - Automatic version identification
 - To avoid removal of these changes

See:
 - [The CI job definition](https://github.com/dash0hq/opentelemetry-collector-web/blob/ce5891a88fe7458d8ce8a8155294f04ae1b920bb/.github/workflows/discover-new-releases.yaml#L68-L115)